change/@microsoft-teams-js-cf7d9559-bf41-4407-bbc2-f1d5cc910a36.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Moving chat.openConversation and chat.closeConversation into a subcategory chat.conversation. Added in new APIs chat.openChat and chat.openGroupChat to leverage deeplinking to open Teams chats with one or more user",
```

Comment:
I might replace the word "leverage" with "replace". These logs are for developers looking to update their apps to use the new functionality. While under the covers we might still be support some deep linking scenarios, I think it makes more sense for app devs to think of this change as a replacement for the teams-specific deep linking model

Reply:
Good idea